code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -690,7 +690,7 @@
 	name = "Monkey"
 	antag_flag = ROLE_MONKEY
 	antag_datum = /datum/antagonist/monkey/leader
-	restricted_roles = list("Cyborg", "AI")
+	restricted_roles = list("Prisoner","Cyborg", "AI")
```

Comment:
make a protected roles list for these

Reply:
Actually i dont think it is required, since monkey gamemode is removed. I put prisoner restriction here just because... yes.